parser_library/src/processing/preprocessors/preprocessor_utils.cpp

```diff
@@ -72,50 +141,56 @@ semantics::preproc_details::name_range get_stmt_part_name_range(
 
 template<typename PREPROC_STATEMENT, typename ITERATOR>
 std::shared_ptr<PREPROC_STATEMENT> get_preproc_statement(
-    const std::match_results<ITERATOR>& matches, stmt_part_ids ids, size_t lineno)
+    const std::match_results<ITERATOR>& matches, stmt_part_ids ids, size_t lineno, size_t continuation_column)
```

Comment:
```suggestion
    const std::match_results<ITERATOR>& matches, const stmt_part_ids& ids, size_t lineno, size_t continuation_column)
```
and make the arguments static const?